gradle.properties

```diff
@@ -1,6 +1,6 @@
 mod_id=skyclientcosmetics
-mod_name=Skyclient Cosmetics
-mod_version=0.7.0-beta
-mod_description=Cosmetics for the mod installer Skyclient!
+mod_name=SkyClient Cosmetics
+mod_version=0.7.05-lisena
```

Comment:
Still wrong version also it won't be prob detected by scu

Reply:
what do you want the version to be, `0.7.1-beta` or `0.7.1-lisena` or what?

Reply:
as I said before you didnt add one dot also just replace lisena with beta